npm-shrinkwrap.json

```diff
@@ -1,70 +1,72 @@
 {
   "name": "perseo",
-  "version": "0.4.0-next",
```

Comment:
I think I have spoted some version changes. However, let me have a nuew look to it and, if find some, I will mark the specific lines.